src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -57,6 +57,9 @@ def load_command_table(self, _):
         g.custom_command('stop', 'app_stop', supports_no_wait=True)
         g.custom_command('restart', 'app_restart', supports_no_wait=True)
         g.custom_command('logs', 'app_tail_log')
+        g.custom_command('identity assign', 'app_identity_assign')
+        g.custom_command('identity remove', 'app_identity_remove')
+        g.custom_command('identity show', 'app_identity_show')

```

Comment:
Suggest to move the code to a new command group scope 'spring-cloud app identity.'

Reply:
use `custom_show_command` for 'identity show'

Reply:
Good point. Will do.